types/has/has-tests.ts

```diff
@@ -0,0 +1,4 @@
+import has = require('has');
+
+// $ExpectType boolean
+has(Object, 'hasOwnProperty');
```

Comment:
```suggestion
// $ExpectType boolean
has(Object.prototype, 'hasOwnProperty');
```
even tho it doesn't matter that it would have returned false, best not to leave a confusing example :-)